tools/spec-gen-sdk/src/automation/workflowPackage.ts

```diff
@@ -174,10 +174,11 @@ const workflowPkgSaveApiViewArtifact = async (context: WorkflowContext, pkg: Pac
   if (!existsSync(destination)) {
     fs.mkdirSync(destination, { recursive: true });
   }
-  context.sdkApiViewArtifactFolder = destination;
   const fileName = path.basename(pkg.apiViewArtifactPath);
-  context.logger.info(`Copy apiView artifact from ${path.join(context.config.localSdkRepoPath, pkg.apiViewArtifactPath)} to ${path.join(destination, fileName)}`);
-  copyFileSync(path.join(context.config.localSdkRepoPath, pkg.apiViewArtifactPath), path.join(destination, fileName));
+  const newApiViewArtifactPath = path.join(destination, fileName);
+  context.logger.info(`Copy apiView artifact from ${path.join(context.config.localSdkRepoPath, pkg.apiViewArtifactPath)} to ${newApiViewArtifactPath}`);
+  copyFileSync(path.join(context.config.localSdkRepoPath, pkg.apiViewArtifactPath), newApiViewArtifactPath);
+  pkg.apiViewArtifactPath = newApiViewArtifactPath;
```

Comment:
I don't see `apiViewArtifactPath` being used in the runner or pipeline yml.

Reply:
It's changed to `apiViewArtifact ` https://github.com/Azure/azure-sdk-tools/blob/d008f93d062cea221e32f2e5d2ac5b4f7feb371b/tools/spec-gen-sdk/src/types/ExecutionReport.ts#L27